manager/controllers/app/select_modules.go

```diff
@@ -352,7 +352,7 @@ func validateModuleRestrictions(item *DataInfo, edge *Edge) bool {
 
 func validateClusterRestrictions(item *DataInfo, edge *ResolvedEdge, cluster multicluster.Cluster) bool {
 	capability := edge.Module.Spec.Capabilities[edge.CapabilityIndex]
-	if !validateClusterRestrictionsPerCapability(item, capability.Capability, cluster) {
+	if !validateClusterRestrictionsPerCapability(item, string(capability.Capability), cluster) {
```

Comment:
Same comment about changing the argument type instead.

Reply:
changed